### PR TITLE
PR for merging content on Notebook for RAG with Llama 3 open-source and Elastic #260

### PR DESCRIPTION
**Pull request by rishikeshradhakrishnan**
Notebook for RAG with Llama 3 open-source and Elastic #260

### CODE REVIEW

File: notebooks/integrations/llama3/rag-elastic-llama3.ipynb [hunk 1]
```diff
@@ -0,0 +1,317 @@
+{
```
Comment (joemcelroy) [inline notebooks/integrations/llama3/rag-elastic-llama3.ipynb:1]: _Commented on notebook `notebooks/integrations/llama3/rag-elastic-llama3.ipynb` Cell 4 Line 1_
```python
!pip install llama-index
```
nitpick: update this to be one line for pip install, similar to how the other notebooks have done this

_Respond and view the context [here](https://gitnotebooks.com/elastic/elasticsearch-labs/pull/266)._

Comment (joemcelroy) [inline notebooks/integrations/llama3/rag-elastic-llama3.ipynb:1]: _Commented on notebook `notebooks/integrations/llama3/rag-elastic-llama3.ipynb` Cell 6 Line 2_
```python
from llama_index.core.node_parser import SentenceSplitter
from llama_index.core.ingestion import IngestionPipeline
```
nitpick: move the imports to where the code being used, so its obvious which cell needs which import.

_Respond and view the context [here](https://gitnotebooks.com/elastic/elasticsearch-labs/pull/266)._

Comment (joemcelroy) [inline notebooks/integrations/llama3/rag-elastic-llama3.ipynb:1]: _Commented on notebook `notebooks/integrations/llama3/rag-elastic-llama3.ipynb` Cell 13 Line 1_
```markdown
## Execute pipeline, 
```
remove ,

_Respond and view the context [here](https://gitnotebooks.com/elastic/elasticsearch-labs/pull/266)._

Comment (joemcelroy) [inline notebooks/integrations/llama3/rag-elastic-llama3.ipynb:1]: _Commented on notebook `notebooks/integrations/llama3/rag-elastic-llama3.ipynb` Cell 15 Line 1_
```markdown
The embeddings are stored in a dense vector field of dimension `4096`. The dimension size comes from the size of the embeddings generated from `Llama3`.
```
is that right? the embedding is 4096 dims? Could we improve this with picking a smaller dim model instead if so.

_Respond and view the context [here](https://gitnotebooks.com/elastic/elasticsearch-labs/pull/266)._

Comment (joemcelroy) [inline notebooks/integrations/llama3/rag-elastic-llama3.ipynb:1]: _Commented on notebook `notebooks/integrations/llama3/rag-elastic-llama3.ipynb` Cell 1 Line 1_
```markdown
# RAG with Elastic and Llama3
```
this examples are either LangChain or Llama-index. I would update it like so and be obvious that its using these frameworks behind the scenes. 

_Respond and view the context [here](https://gitnotebooks.com/elastic/elasticsearch-labs/pull/266)._

Comment (rishikeshradhakrishnan) [reply notebooks/integrations/llama3/rag-elastic-llama3.ipynb:1]: Updated the code.

Comment (rishikeshradhakrishnan) [reply notebooks/integrations/llama3/rag-elastic-llama3.ipynb:1]: Updated it accordingly.

Comment (rishikeshradhakrishnan) [reply notebooks/integrations/llama3/rag-elastic-llama3.ipynb:1]: Updated this as well.

Comment (rishikeshradhakrishnan) [reply notebooks/integrations/llama3/rag-elastic-llama3.ipynb:1]: AFAIK and checked, there didnt seem to be any option to change that.

Comment (rishikeshradhakrishnan) [reply notebooks/integrations/llama3/rag-elastic-llama3.ipynb:1]: Updated it.

**Review by joemcelroy — APPROVED** [review]

**Pull request merged**